### PR TITLE
Replacing Custom Oses with versionless rids

### PR DESCRIPTION
**Pull request by Anipik**
Relate runtime PR https://github.com/dotnet/runtime/pull/33315

This change is required to enable to pass the win formatting leg.

### CODE REVIEW

File: doc/cijobs.md [hunk 1]
```diff
@@ -65,7 +65,7 @@ The "cijobs copy" command has the following help message:
 
 ## Using cijobs
 
-A common question for a developer might be "what is the last successful OSX checked build?"
+A common question for a developer might be "what is the last successful  checked build?"
```
Comment (ericstj) [inline doc/cijobs.md]: Looks broken

File: sample_config.json [hunk 2]
```diff
@@ -1,37 +1,37 @@
...
-            "core_root": "c:\\gh\\coreclr\\bin\\tests\\Windows_NT.x86.checked\\Tests\\Core_Root",
-            "test_root": "c:\\gh\\coreclr\\bin\\tests\\Windows_NT.x86.Release"
+            "core_root": "c:\\gh\\coreclr\\bin\\tests\\win.x86.checked\\Tests\\Core_Root",
+            "test_root": "c:\\gh\\coreclr\\bin\\tests\\win.x86.Release"
         },
         "tools": [
             {
                 "tag": "checked_osx-1439",
-                "path": "/Users/russellhadley/Work/glue/output/tools/checked_osx-1439/Product/OSX.x64.Checked"
+                "path": "/Users/russellhadley/Work/glue/output/tools/checked_osx-1439/Product/.x64.Checked"
```
Comment (ericstj) [inline sample_config.json]: Again broken

**Comment by Anipik** [comment]
@ericstj  i suggest  merging this after the runtime PR, bcoz otherwise it will lead to all PRs in the runtime having failed formatting job. or maybe we can merge both of them together.

**Comment by ViktorHofer** [comment]
What about other release branches (5.0 and 3.x) which consume jitutils from master?

File: doc/cijobs.md [hunk 3]
```diff
@@ -65,7 +65,7 @@ The "cijobs copy" command has the following help message:
 
 ## Using cijobs
 
-A common question for a developer might be "what is the last successful OSX checked build?"
+A common question for a developer might be "what is the last successful OSX  checked build?"
```
Comment (ericstj) [inline doc/cijobs.md]: nit: stray space
```suggestion
A common question for a developer might be "what is the last successful OSX checked build?"
```

File: bootstrap.sh [hunk 4]
```diff
@@ -12,32 +12,32 @@ function get_host_os {
...
             ;;
 
         NetBSD)
-            __HostOS=NetBSD
+            __HostOS=netbsd
             ;;
 
         SunOS)
-            __HostOS=SunOS
+            __HostOS=sunos
```
Comment (ericstj) [inline bootstrap.sh]: Haven't seen this RID before.  Is it going to be a problem that we don't have a mapping for this in runtime.json?

**Comment by ericstj** [comment]
The diff looks reasonable to me, but I can't really sign off on this as I don't know the purpose of most of these files.

**Comment by BruceForstall** [comment]
Closing this since https://github.com/dotnet/runtime/pull/33315 was closed.

**Pull request closed without merging**